### ISSUE TITLE
Change from: email to info@sfwater.org for automated emails

### ISSUE THREAD

**Issue by jasonlally** [sfbrigade/adopt-a-drain#91]
@jszwedko is it as simple as just changing this:

```ruby
class ApplicationMailer < ActionMailer::Base
  default from: 'adoptadrain@codeforsanfrancisco.org'
  layout 'mailer'
end
```
in application_mailer.rb

I wasn't sure if there was something else that needed to happen to make this change.

**Comment by jszwedko** [comment]
@jasonlally yeah, that should be it -- I don't think we are doing anything fancier like publishing a SPF record for the domain.

**Comment by jszwedko** [comment]
It looks like we are publishing one for `sfwater.org` though:

```
➜  adopt-a-drain git:(production) nslookup -type=txt sfwater.org
Server:		192.168.1.1
Address:	192.168.1.1#53

Non-authoritative answer:
sfwater.org	text = "v=spf1 a mx ~all"
```

We may want to add `sendgrid.net` there (see https://sendgrid.com/docs/Glossary/spf.html) to reduce the probability it'll be gray listed or marked as spam.

**Comment by jszwedko** [comment]
Closed by #95
